spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -142,6 +142,8 @@ func TestSample(t *testing.T) {
 	mustRunSample(t, createDatabase, dbName, "failed to create a database")
 	runSample(t, createClients, dbName, "failed to create clients")
 	runSample(t, write, dbName, "failed to insert data")
+	out = runSample(t, batchWrite, dbName, "failed to write data using BatchWrite")
+	assertNotContains(t, out, "could not be applied with error")
```

Comment:
nit: can we instead check for the success case `have been applied with commit timestamp`?

Reply:
Since there can be partial success, the code checks if there was any failure.